src/shared/bootspec.c

```diff
@@ -1297,13 +1427,108 @@ static void boot_entry_file_list(
                 *ret_status = status;
 }
 
+static void print_addon(
+                BootEntryAddon *addon,
+                const char *addon_str) {
+
+        printf("  %s: %s\n", addon_str, addon->location);
+        printf("      cmdline: %s%s\n", special_glyph(SPECIAL_GLYPH_TREE_RIGHT), addon->cmdline);
+}
+
+static int print_cmdline(const BootEntry *e) {
+        _cleanup_free_ char *final_cmdline = NULL;
```

Comment:
I disagree with this. I think it's easier to provide the user the full line instead of leaving exercises to the reader. It doesn't cost us nothing, but instead if one has many addons it could become handy to have it all in one place.

Reply:
i find it redundant, but I also don't mind as long as it is suppressed in the "Pretty" outut if identical to the regular cmdline. let's not confuse users please

also call it "combined", not "final" cmdline

Reply:
and please stick to the "options" moniker in the output. We inherited that from the type1 bls spec, we should stick to it i think

Reply:
Agree in removing it if it's identical to regular cmdline. 
Just one thing: "combinedcmdline" is too long. The whole indentation of strings will be weird:
```
Default Boot Loader Entry:
         type: Boot Loader Specification Type #2 (.efi)
        title: Fedora Linux 40 (Server Edition Prerelease)
           id: devel.efi
       source: /boot/efi//EFI/Linux/devel.efi
     sort-key: fedora
      version: 40 (Server Edition Prerelease)
        linux: /boot/efi//EFI/Linux/devel.efi
      options: console=tty0 console=ttyS0 quiet loglevel=3 systemd.show_status=>
  globaladdon: loader/addons/rpm_addon.addon.efi
      options: └─this is a normal addon
  globaladdon: loader/addons/global.addon.efi
      options: └─quiet loglevel=3 systemd.show_status=auto
   localaddon: devel.efi.extra.d/rpm_addon.addon.efi
      options: └─this is a normal addon
combinedcmdline: console=tty0 console=ttyS0 quiet loglevel=3
```
Ideas?

Reply:
How about just "cmdline", given it's the effective one being passed?
But also adjusting indentation of the previous items is also fine, the exact whitespace formatting is not an API

Reply:
for me it's fine, but I'll let you decide

Reply:
Either works for me. @poettering ?

Reply:
i'd also be fine with `comb.opt.:` or so.

Reply:
but cmdline is ok by me too, as long as the original field keeps its name

Reply:
cmdline it is